marathon/adapter_test.go

```diff
@@ -101,6 +112,25 @@ func (c *mockConverter) convertToApp(service *api.Service) *gomarathon.Applicati
 	return args.Get(0).(*gomarathon.Application)
 }
 
+// Mock the Deployer
+type mockDeployer struct {
+	mock.Mock
+	deployStatus status
+}
+
+func (d *mockDeployer) setStatus(s status) {
+	d.deployStatus = s
```

Comment:
If possible we should keep the mock object similar to the real object, or it does not represent the real object anymore. Do we really need the ```setStatus``` method?

Also, can you add a test that checks that the MarathonDeployer implements the DeployerInterface? This will alert us if the interface is broken.